src/ipc/ipc3/dai.c

```diff
@@ -58,6 +58,18 @@ int dai_config_dma_channel(struct comp_dev *dev, void *spec_config)
 		handshake = dai_get_handshake(dd->dai, dai->direction,
 					      dd->stream_id);
 		channel = EDMA_HS_GET_CHAN(handshake);
+#if CONFIG_IMX8ULP
+		switch (dai->dai_index) {
+		case 5:
+			imx8ulp_dmamux_rx_int = IMX8ULP_DMAMUX2_SAI5_RX;
+			imx8ulp_dmamux_tx_int = IMX8ULP_DMAMUX2_SAI5_TX;
+			break;
+		case 6:
+			imx8ulp_dmamux_rx_int = IMX8ULP_DMAMUX2_SAI6_RX;
+			imx8ulp_dmamux_tx_int = IMX8ULP_DMAMUX2_SAI6_TX;
+			break;
+		}
```

Comment:
do we need a default here to warn users that config is bad ? @dbaluta ?

Reply:
Sorry, I did not see this was going in generic dai.c. Can we get this information elsewhere, I think this is statically encoded for Intel SSP and DMICs (so could be copied ?).